routers/web/repo/issue.go

```diff
@@ -1020,32 +1020,40 @@ func commentTag(repo *models.Repository, poster *models.User, issue *models.Issu
 	if err != nil {
 		return models.CommentTagNone, err
 	}
-	if perm.IsOwner() {
-		if !poster.IsAdmin {
-			return models.CommentTagOwner, nil
-		}
 
-		ok, err := models.IsUserRealRepoAdmin(repo, poster)
-		if err != nil {
-			return models.CommentTagNone, err
-		}
+	// By default the poster has no tags.
+	pCommentTag := models.CommentTagNone
```

Comment:
`pCommentTag` what does `p` mean? Maybe we can have a full variable name here.

Reply:
It means `pointer` to a `CommentTag`.

Reply:
`type CommentTag int, CommentTagNone CommentTag = iota`, how can `pCommentTag` be a pointer.

Reply:
My fault - `p` meant posterCommentTag.

But it think it can just be `commentTag`